src/libraries/System.Security.Cryptography.Encoding/src/Internal/Cryptography/AsnFormatter.cs

```diff
@@ -48,11 +46,9 @@ private static string EncodeHexString(byte[] sArray, uint start, uint end, bool
                         hexOrder[j++] = ' ';
                     }
 
-                    uint digit = (uint)((sArray[i] & 0xf0) >> 4);
-                    hexOrder[j++] = s_hexValues[digit];
-
-                    digit = (uint)(sArray[i] & 0x0f);
-                    hexOrder[j++] = s_hexValues[digit];
+                    int digit = sArray[i];
+                    hexOrder[j++] = HexConverter.ToCharUpper (digit >> 4);
+                    hexOrder[j++] = HexConverter.ToCharUpper (digit);
```

Comment:
Some unexpected spaces got inserted between the method name and the parentheses here.

Reply:
Fixed